modules/acquia_cms_headless/src/Plugin/AcquiaCmsHeadless/HeadlessApiUsers.php

```diff
@@ -245,7 +253,7 @@ public function buildEntityRows(): array {
       $row = [
         'name' => Link::fromTextAndUrl($user->label(), $operations[$user->id()]['edit']['url']),
         // Currently only displaying users assigned the headless role.
-        'role' => $this->t('Headless Role'),
+        'role' => $this->headlessRoleLabel,
```

Comment:
@deepakmishra2 why we removed t() method?

Reply:
Because its a variable now and the entity which we are pulling would pull translated content.

Reply:
@deepakmishra2 Are you sure? Because variable isn't loading the entity on the basis of langcode?

cc: @vishalkhode1